string robot_name
geometry_msgs/PoseStamped pose
---
bool success